drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/RightPanelView.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
```

Comment:
2018

Reply:
Fixed with
[DROOLS-2780] Refactored RightPanelView/RightPanel interaction. Removing access modifier for testing purpose. Renaming mocked instances.